service.subtitles.pipocas/default.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Service Pipocas.tv
+
+#simple call, it's just a subtitle addon
+from service import Main
+
+#run the program
+Main().get_params
```

Comment:
are you sure this works? `get_params` is not annotated as a `@property` so I doubt you can run the method like this. Missing ()?

```suggestion
Main().get_params()
```

Reply:
It does work :) tested but I'll change it.

Reply:
done